analysis/Scripts/Imports.py

```diff
@@ -24,10 +27,11 @@ def getDataCuts(run, trig = False):
 		if trig:
 			trigger_cuts = "lcplus_L0HadronDecision_TOS == 1 && lcplus_Hlt1TrackAllL0Decision_TOS == 1 && lcplus_Hlt2CharmHadD2HHHDecision_TOS == 1"
 		else:
-			trigger_cuts = "lcplus_Hlt2CharmHadD2HHHDecision_TOS ==1"
+			trigger_cuts = ""
 	elif run == 2:
 		if trig:
-			trigger_cuts = "lcplus_L0HadronDecision_TOS == 1 && lcplus_Hlt1TrackMVADecision_TOS == 1 && (lcplus_Hlt2CharmHadXicpToPpKmPipTurboDecision_TOS == 1 || lcplus_Hlt2CharmHadLcpToPpKmPipTurboDecision_TOS == 1)"
+			#trigger_cuts = "lcplus_L0HadronDecision_TOS == 1 && lcplus_Hlt1TrackMVADecision_TOS == 1 && (lcplus_Hlt2CharmHadXicpToPpKmPipTurboDecision_TOS == 1 || lcplus_Hlt2CharmHadLcpToPpKmPipTurboDecision_TOS == 1)"
+			trigger_cuts = "lcplus_L0HadronDecision_TOS == 1 && lcplus_Hlt1TrackAllL0Decision_TOS == 1 && lcplus_Hlt2CharmHadD2HHHDecision_TOS == 1"
```

Comment:
Should this not be `lcplus_Hlt1TrackMVADecision_TOS == 1` instead? As it is for run 2?

Reply:
Yes as you can see in Ln 33. Sorry I broke your code.